app/web/src/features/auth/email/ChangeEmail.tsx

```diff
@@ -11,16 +11,24 @@ import { useMutation } from "react-query";
 import { service } from "service";
 import { lowercaseAndTrimField } from "utils/validation";
 
+import makeStyles from "../../../utils/makeStyles";
```

Comment:
use absolute imports for things outside this feature (see the line above this one)

Reply:
Changed it in last push